src/ovirtapi/transform.js

```diff
@@ -260,20 +260,24 @@ const VmStatistics = {
       memory: {},
       cpu: {},
       network: {},
+      disks: {},
     }
 
     for (const stat: ApiVmStatisticType of statistics) {
       if (stat.kind !== 'gauge') continue
 
       // no values -> undefined, 1 value -> value.datum, >1 values -> [...values.datum]
+      // ?disks.usage -> {detail...}
       const datum =
         stat.values &&
         stat.values.value &&
-        (stat.values.value.length === 1
-          ? stat.values.value[0].datum
-          : stat.values.value.map(value => value.datum))
+        (stat.name === 'disks.usage'
+          ? JSON.parse(stat.values.value[0].detail || 'null')
```

Comment:
I recommend moving the JSON parsing down lower and have it `convertInt()` values here so the `parseInt` doesn't need to be done in the components.

````js
if (stat.name === 'disks.usage') {
  datum = JSON.parse(datum)
  datum.total = convertInt(datum.total)
  datum.used = convertInt(datum.used)
}
````